foursight_core/es_connection.py

```diff
@@ -255,7 +271,8 @@ def list_all_keys(self):
         }
         search = Search(using=self.es, index=self.index)
         search.update_from_dict(doc)
-        return self.search(search, key='_id')
+        result, total = self.search(search, key='_id')
+        return result
```

Comment:
I'm won't make a huge fuss about the use of `_`, which is quite normal, but I want to at least note that I often prefer to name the result variables and mark them ignored because it's often useful to call out the value that is available for free if someone wanted it. If you don't put the name, people sometimes recompute it (which may be expensive or fail to do useful synchronization, depending on the function).

Reply:
Sounds good, from both, but taking Kent's suggestion - naming it `unused_total` (or is `ignored_total preferred`?).

Reply:
You can keep total and used the `dcicutils.misc_utils.ignored` method to "ignore" it.

Reply:
All set.